examples/single_column_nutrients_plankton_bacteria_detritus.jl

```diff
@@ -9,74 +9,93 @@ using Oceananigans
 using Oceananigans.Units
 using Oceananigans.TurbulenceClosures: CATKEVerticalDiffusivity
 using Printf
-using CairoMakie
+using GLMakie #CairoMakie
```

Comment:
Once we're ready to merge, convert this back to `CairoMakie` and I think the docs will compile